File: Java/1007 - Diferença.java
/* 
   ==UserScript==
 @name         Diferença
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1007 - Diferença
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/


import java.util.Scanner;
import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scan = new Scanner(System.in);
		int A = scan.nextInt();
		int B = scan.nextInt();
		int C = scan.nextInt();
		int D = scan.nextInt();
		
		int diferenca = A * B - C * D;
		
		System.out.println("DIFERENCA = " + diferenca);
 
    }
 
}

File: C#/1007 - Diferença.cs
/* 
   ==UserScript==
 @name         Diferença
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1007 - Diferença
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

using System;

class URI {
    
  static void Main(string[] args) {
      
    int valueA = Convert.ToInt32(Console.ReadLine());
    int valueB = Convert.ToInt32(Console.ReadLine());
    int valueC = Convert.ToInt32(Console.ReadLine());
    int valueD = Convert.ToInt32(Console.ReadLine());
    
    int diferenca = valueA*valueB - valueC*valueD;
    
    Console.WriteLine("DIFERENCA = " + diferenca);
    
  }
}
